src/services/auth-services.js

```diff
@@ -1,14 +1,47 @@
 import { getDoc, doc } from 'firebase/firestore';
-import { getAuth, createUserWithEmailAndPassword, signInWithEmailAndPassword } from 'firebase/auth';
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+  signInWithPopup,
+  GoogleAuthProvider,
+  signOut,
+} from 'firebase/auth';
 import { db } from '../firebase';
 
-export async function findUser(email, password) {
-  const auth = getAuth();
+const auth = getAuth();
+const provider = new GoogleAuthProvider();
+
+export async function singInEmailAndPassword(email, password) {
   try {
     const {
       user: { uid },
     } = await signInWithEmailAndPassword(auth, email, password);
 
+    return findUser(uid);
+  } catch (error) {
+    console.log(error);
+
+    return false;
```

Comment:
no need to return false here, it's meaningless, remove it everywhere. You should throw Error to catch it on other level or return undefined if you need to return smth

Reply:
answered above